### ISSUE TITLE
Source of current.png

### ISSUE THREAD

**Issue by nurupo** [ItsDuke/Tox-UI#13]
Is there a source file of  [current.png](/ItsDuke/Tox-UI/blob/master/current.png) somewhere?

**Comment by ItsDuke** [comment]
@nurupo yeah, it's a giant adobe illustrator file with a bunch of different icons, colors, and images, and different currently unused variations

**Comment by nurupo** [comment]
Not seeing it in the repository though. Any chance you could add it?

**Comment by ItsDuke** [comment]
I haven't added it to the repository because it's in no shape to share. It's full of half-ideas, unused versions, and just little bit and pieces here and there.
